src/Command/Ingester/AssignmentIngesterCommand.php

```diff
@@ -67,7 +67,30 @@ protected function configure(): void
     {
         parent::configure();
 
-        $this->setDescription('Assignment data ingestion');
+        $this->setDescription('Ingests assignments into the application');
+        $this->setHelp(<<<'EOF'
+The <info>%command.name%</info> command ingests assignments into the application.
```

Comment:
Suggestion: Is it possible to have it in a file something like $this->setHelp(file_get_contents('file'))?
I believe that it'd improve code readability.

Reply:
Not a big fan of the idea to be honest. I think it's better to have the text here rather than in a separate file. But I admit it's not beautiful :/

Reply:
ok. not a problem for me, just a suggestion :)